server/lorax_server/utils/torch_utils.py

```diff
@@ -13,7 +13,7 @@ def is_bf16_supported() -> bool:
 def is_fp8_quantized(config, layer_name):
     # check if quantization is fp8 and either of the fused layers is not ignored
     # typically, either all qkv will be quantized or none so just check for one
-    if config.quantize == "fp8" and hasattr(config, "quantization_config"):
+    if config.quantize and config.quantize.startswith("fp8") and hasattr(config, "quantization_config"):
```

Comment:
I would also make this a utility function. Reason being: we might not continue to adhere to this naming convention in the future.

Reply:
This is actually dead code. Removed it.